.travis.yml

```diff
@@ -57,31 +54,28 @@ python:
 
 before_install:
   - sudo apt-get install enchant shellcheck
+  - echo $maxikube_secret_pass | gpg --passphrase-fd 0 maxikube_secrets.tar.gpg
+  - tar xvf maxikube_secrets.tar
+  - export DOCKER_CERT_PATH="$PWD/docker-certs"
+  - mkdir -p ~/.kube
+  - mv admin.conf ~/.kube/config
 
 install:
   - ./scripts/travis-install.sh
 
-before_script: |-
-    make start
+before_script:
+  # Use a unique name to deploy renku
+  - export RENKU_DEPLOY="renku-$(openssl rand -hex 4)"; echo $RENKU_DEPLOY
+  - export MAXIKUBE_HOST=$(kubectl get nodes minikube -ojson | jq -r '.status.addresses[] | select(.type | contains("InternalIP")) | .address'); echo $MAXIKUBE_HOST
+  - ./scripts/travis-before-script.sh
 
 script:
-  - xvfb-run make test
-  # TODO - helm lint charts/renku -f charts/minikube-values.yaml --set renku.domain=renku.build
-
-before_deploy:
-  # install chartpress
-  - pip install -U pip
-  - pip install -e git+https://github.com/jirikuncar/chartpress#egg=chartpress
-  # install Helm
-  - wget -q ${HELM_URL}/${HELM_TGZ}
-  - tar xzfv ${HELM_TGZ}
-  - PATH=`pwd`/linux-amd64/:$PATH
-  - helm init --client-only
+  - ./scripts/travis-build.sh
+  - ./scripts/run-tests.sh
 
 deploy:
   - provider: script
     skip_cleanup: true
     script: bash scripts/travis-deploy.sh
     on:
       branch: master
-      condition: $TRAVIS_EVENT_TYPE != cron
```

Comment:
is there a reason why this was removed here?

Reply:
But it works now! :rofl: 
I guess the PR was based before the line was introduced

Reply:
I just noticed lots of pushes to the helm charts at a 24-hour cadence...